modules/RTC/TPCUtils.js

```diff
@@ -401,11 +377,29 @@ export class TPCUtils {
         const simulcastLine = browser.isFirefox() && browser.isVersionGreaterThan(71)
             ? `recv ${SIM_LAYER_RIDS.join(';')}`
             : `recv rid=${SIM_LAYER_RIDS.join(';')}`;
+        const sdp = transform.parse(desc.sdp);
+        const mLines = sdp.media.filter(m => m.type === MediaType.VIDEO);
+        const senderMids = Array.from(this.pc._localTrackTransceiverMids.values());
```

Comment:
Do we need `Array.from`? The return value from Map.values should be iterator which also has find method. Wouldn't that work?

Reply:
Technically its not needed, added here for clarity. I don't see any harm in using it though :)

Reply:
Generally for performance? We are adding a redundant operation. 

Also I don't understand the clarity argument. Actually for me it was the opposite I started wondering what would the .values() call return so that we need to convert it to array. Would you explain what the array conversation clarifies in this case?

Reply:
Actually that doesn't seem to be true. `values()` returns a Map iterator which doesn't support find.

Reply:
I was looking into https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Iterator/find . But just gave it a try and it seems to not work for some reason...